dora/integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -176,16 +179,23 @@ private void umountInternal() {
       }
     } else {
       try {
-        exitCode = new ProcessBuilder("fusermount", "-u", "-z", mountPath).start().waitFor();
+        Pair<Integer, String> output = Environment.runCommandAndCapture(
```

Comment:
nit: I'm not a big fan to use Pair,  it might be better to declare a class for the result explicitly

Reply:
yeah this entire func is not used any more, these are some codes for debugging purposes. let me remove these changes.